### PR TITLE
Add deeplink above code block that links to the API reference for each request in the console

### LINKED ISSUE #135: Add deeplink to the docs for each request in the console
I mostly use the auto completion to write the endpoint I want to query in the console but then I do not remember all the parameters which are required / optional. I just want to click on a link which deeplinks to the endpoint I want to query. The link could be next to the "RUN | BEAUTIFY" buttons. What do you think?

### PR DESCRIPTION
**Pull request by Snehil-Shah**
Closes #135 

Added a "DOCS" button next to the "RUN | BEAUTIFY" buttons that directly links to that **Specific** Request's API reference.

## Features

- **Quick insert commands and access the docs**:
insert

- **Dynamic Nature**: Only visible for valid commands, also works for endpoints with path variables
dynamic

### CODE REVIEW

**Review by kartik-gupta-ij — COMMENTED** [review]
Hey,
I think it would be better if you shift the logic to [autocomplete-openapi](https://github.com/qdrant/autocomplete-openapi)

Ideas :
1. That will return auto-completes and link.
-------------------or--------------------
2. That will return auto-completes and auto write the required json.

**Comment by Snehil-Shah** [comment]
Hey, are you talking about autocompleting the request body?

**Comment by kartik-gupta-ij** [comment]
Yes, only those fields which are mandatory.

**Comment by Snehil-Shah** [comment]
@kartik-gupta-ij can I contact you on Discord regarding this? I have some ideas and doubts regarding this.

**Comment by kartik-gupta-ij** [comment]
> @kartik-gupta-ij can I contact you on Discord? I have some ideas and doubts regarding this.

yeah sure!

**Comment by Snehil-Shah** [comment]
> > @kartik-gupta-ij can I contact you on Discord? I have some ideas and doubts regarding this.
> 
> yeah sure!

DM'd you!

**Comment by Snehil-Shah** [comment]
Depends on https://github.com/qdrant/autocomplete-openapi/pull/3

**Comment by Snehil-Shah** [comment]
@kartik-gupta-ij `import.meta.env` is giving error when running in tests. how can I fix this?

File: src/components/EditorCommon/config/Rules.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
+import { OpenapiDocs } from 'autocomplete-openapi/src/request-docs';
+
 const Method = ['POST', 'GET', 'PUT', 'DELETE', 'PATCH', 'HEAD'];
 
+const response = await fetch(import.meta.env.BASE_URL + './openapi.json');
+const openapi = await response.json();
```
Comment (kartik-gupta-ij) [inline src/components/EditorCommon/config/Rules.js:6]:
```suggestion
import { OpenapiDocs } from 'autocomplete-openapi/src/request-docs';
import { readFile } from 'fs/promises';

const Method = ['POST', 'GET', 'PUT', 'DELETE', 'PATCH', 'HEAD'];

let openapi;
if (import.meta.env.MODE === 'test') {
  const response = await readFile('public/openapi.json', 'utf8');
  openapi = JSON.parse(response);
} else {
  const response = await fetch(import.meta.env.BASE_URL + 'openapi.json');
  openapi = await response.json();
}
```
@Snehil-Shah, When running tests, vitest does not serve local files, resulting in the error you encountered.

**Review by kartik-gupta-ij — CHANGES_REQUESTED** [review]

**Review by kartik-gupta-ij — APPROVED** [review]

**Pull request merged**